crates/storage/src/lazy/lazy_hmap.rs

```diff
@@ -54,6 +54,9 @@ use ink_prelude::{
 };
 use ink_primitives::Key;
 
+#[cfg(doc)]
+use crate::lazy::LazyIndexMap;
+
```

Comment:
Why is this needed?

Reply:
Same as above.

Reply:
I think I'd rather want to write-out the path entirely for the private docs instead of having another import statement that is potentially confusing other people just as me.